platform/darwin/src/NSArray+MGLAdditions.mm

```diff
@@ -23,4 +23,29 @@ @implementation NSArray (MGLAdditions)
     return vector;
 }
 
++ (NSArray *)mgl_coordinatesFromCoordinates:(std::vector<CLLocationCoordinate2D>)coords {
```

Comment:
Let's categorize NSCoder to have -encodeMGLLocationCoordinate2D:forKey: and -decodeMGLLocationCoordinate2DForKey:, as has been done for other structs like CGSize and UIEdgeInsets.